libs/editor/WordPressEditor/src/main/java/org/wordpress/android/editor/GutenbergEditorFragment.java

```diff
@@ -378,10 +379,12 @@ public void run() {
 
     private ArrayList<MediaOption> initOtherMediaImageOptions() {
         ArrayList<MediaOption> otherMediaOptions = new ArrayList<>();
+        FragmentActivity activity = getActivity();
```

Comment:
AFAIU the surrounded code, the editor will be opened, but the "stock photos" option will be missing. I think we might want to log this into a log (maybe even sentry). Wdyt?

Reply:
I don't think it will be opened if the activity is null though but I can add tracking in case something else crashes after it passes this check 👍